### ISSUE TITLE
use case: create level from code and write to file correct

### ISSUE THREAD

**Issue by Folleach** [Folleach/GeometryDashAPI#18]
May looks like this
```cs
var level = new Level();
// some manipulation with level

var local = new LocalLevels();
local.Add(level);
local.Save();
```

**Comment by Folleach** [comment]
There are an api
```cs
var manager = GameManager.CreateNew();
manager.PlayerName = "Folleach";
manager.PlayerCube = 2;
manager.PlayerShip = 3;

var local = LocalLevels.CreateNew();
local.AddLevel(LevelCreatorModel.CreateNew("haha", manager, new Level()
{
    Blocks =
    {
        new BaseBlock(1)
        {
            PositionX = 15,
            PositionY = 30
        }
    }
}));

var yetAnother = LevelCreatorModel.CreateNew("yet another", manager, new Level()
{
    Blocks =
    {
        new TextBlock()
        {
            PositionX = 90,
            PositionY = 90,
            Text = "yet another"
        }
    }
});
yetAnother.Revision = 999;
yetAnother.Description = "hello";
local.AddLevel(yetAnother);

await local.SaveAsync();
await manager.SaveAsync();
```

<table>
  <tr>
    <td>
      <img src="https://github.com/Folleach/GeometryDashAPI/assets/32067915/a4739499-3a44-4eeb-9ccc-814272c1a3e7" />
    </td>
    <td>
      <img src="https://github.com/Folleach/GeometryDashAPI/assets/32067915/09031348-4870-4aa7-86cd-f63b760bba8c" />
    </td>
  </tr>
</table>

**Comment by Folleach** [comment]
Released in [v0.2.8](https://www.nuget.org/packages/GeometryDashAPI/0.2.8)
